src/components/Tabs/Tabs.js

```diff
@@ -1,10 +1,16 @@
 import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 
+import { range } from '../../util/fp';
+
 import TabList from './components/TabList';
 import Tab from './components/Tab';
 import TabPanel from './components/TabPanel';
 
+const KEY_LEFT_ARROW = 37;
+const KEY_RIGHT_ARROW = 39;
+const KEY_DOWN_ARROW = 40;
```

Comment:
Do you think it would make sense to add these to the [key-helper util](https://github.com/sumup/circuit-ui/blob/master/src/util/key-codes.js)?